pan3d/dataset_viewer.py

```diff
@@ -36,6 +36,7 @@ def __init__(
         server: Union[Server, str] = None,
         state: dict = None,
         pangeo: bool = False,
+        esgf: bool = False,
```

Comment:
are we expecting to have more catalog types?
Maybe replace pangeo+esgf with a single variable `catalogs={'pangeo', 'esgf', ...}`

Reply:
Ok, I swapped out the multiple boolean kwargs for one list kwarg in [36683c2](https://github.com/Kitware/pan3d/pull/67/commits/36683c226871adcb2d6da6dc9792177390ac21f8).